src/internal/version/restore.go

```diff
@@ -0,0 +1,20 @@
+package version
+
+import "math"
+
+const (
```

Comment:
I like having the comments on these. A few small non-blocking nits though:
* comments don't note that they actually apply to OneDrive stuff
* splitting restore and backup versions is a bit confusing. They're actually related to each other but having them different like this makes me think they refer to different things

Reply:
> comments don't note that they actually apply to OneDrive stuff

Added

> splitting restore and backup versions is a bit confusing. They're actually related to each other but having them different like this makes me think they refer to different things

They were already split in the code, this is just migration.  We can make a tech-debt follow-up to unify them if needed.

Reply:
Ah, as soon as I looked at the code I grokked what you were saying.  Sorry, my brain is all a scatter today.  I've made a pass at the locations and naming scheme for the labels.  Should hopefully fit what you're looking for.